packages/events/src/util.ts

```diff
@@ -56,11 +57,21 @@ export function getEventServiceUrl(
         // pick the highest access level that this user has access to
         result = obj.admin || obj.org || /* istanbul ignore next */ obj.public;
       }
-      // single-layer service
-      let url = `${result.url}/0`;
-      // force https
-      url = url.replace(/^http:/gi, "https:");
-      return url;
+      if (result && result.url) {
+        const host = getHubUrl(requestOptions);
+
+        // Extract the Event service's view name; the view returned depends
+        // on permission level of request user
+        const view = result.url.match(/services\/(.*?)\/FeatureServer/);
+
+        // Generate a root url for the hub-indexer event routes
+        /* istanbul ignore next */
```

Comment:
just curious, why the `/* istanbul ignore next */`?

Reply:
> just curious, why the `/* istanbul ignore next */` here?

Great catch! I meant to use `else`. Fixed it now